cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TemporalTest.scala

```diff
@@ -538,37 +542,64 @@ class TemporalTest extends DocumentingTest {
             |[options="header"]
             ||===
             || Component      | Component Group | Description | Type
+            || `duration.quartersOfYear` | Months | The number of _quarters_ in the group that do not make a whole _year_ | Integer
             || `duration.monthsOfYear` | Months | The number of _months_ in the group that do not make a whole _year_ | Integer
-            || `duration.minutesOfHour` | Seconds | The total number of _minutes_ in the group that do not make a whole _hour_ | Integer
-            || `duration.secondsOfMinute` | Seconds | The total number of _seconds_ in the group that do not make a whole _minute_ | Integer
-            || `duration.millisecondsOfSecond` | Seconds | The total number of _milliseconds_ in the group that do not make a whole _second_ | Integer
-            || `duration.microsecondsOfSecond` | Seconds | The total number of _microseconds_ in the group that do not make a whole _second_ | Integer
-            || `duration.nanosecondsOfSecond` | Seconds | The total number of _nanoseconds_ in the group that do not make a whole _second_ | Integer
+            || `duration.monthsOfQuarter` | Months | The number of _months_ in the group that do not make a whole _quarter_ | Integer
+            || `duration.daysOfWeek` | Days | The number of _days_ in the group that do not make a whole _week_ | Integer
+            || `duration.minutesOfHour` | Seconds | The number of _minutes_ in the group that do not make a whole _hour_ | Integer
+            || `duration.secondsOfMinute` | Seconds | The number of _seconds_ in the group that do not make a whole _minute_ | Integer
+            || `duration.millisecondsOfSecond` | Seconds | The number of _milliseconds_ in the group that do not make a whole _second_ | Integer
+            || `duration.microsecondsOfSecond` | Seconds | The number of _microseconds_ in the group that do not make a whole _second_ | Integer
+            || `duration.nanosecondsOfSecond` | Seconds | The number of _nanoseconds_ in the group that do not make a whole _second_ | Integer
```

Comment:
Maybe adjust the column widths here to prevent the first column from spreading over two lines?